vaadin-time-picker-flow/src/main/java/com/vaadin/flow/component/timepicker/TimePicker.java

```diff
@@ -196,6 +220,13 @@ public String getPlaceholder() {
     @Override
     public void setRequired(boolean required) {
         super.setRequired(required);
+        this.required = required;
+    }
+
+    @Override
+    public void setRequiredIndicatorVisible(boolean requiredIndicatorVisible) {
+        super.setRequiredIndicatorVisible(required);
+        this.required = requiredIndicatorVisible;
```

Comment:
BTW, this could a breaking change as we are changing the behaviour of public API.
If somebody was using this indicator without setting `required` to the field (if there is such a case at all), it will be broken now.

Reply:
Seems to be doing the same thing.